frontend/features/contacts/Contacts.tsx

```diff
@@ -15,10 +13,10 @@ export default function Contacts() {
   const selectedContact = useSelector((state: RootState) => state.contacts.selected);
   const filter = useSelector((state: RootState) => state.contacts.filterText);
 
-  const filterChanged = (e: Event) => dispatch(updateFilter((e as TextFieldChangeEvent).target.value));
+  const filterChanged = (e: TextFieldElement.TextFieldValueChangedEvent) => dispatch(updateFilter(e.detail.value));
 
   // TODO: fix event type
-  const handleGridSelection = (e: GridActiveItemChangedEvent<any>) => {
+  const handleGridSelection = (e: GridElement.GridActiveItemChangedEvent<any>) => {
```

Comment:
Not sure if this is better, wdyt? Re-exporting enables the developer to import necessary types from `react-vaadin-components` and to avoid adding dependencies to `@vaadin/vaadin-grid` etc